src/extensions/action-handlers/zapier-action.js

```diff
@@ -71,9 +71,80 @@ export async function onTagUpdate(
 
 // What happens when a texter saves the answer that triggers the action
 // This is presumably the meat of the action
-export async function processAction() {
+export async function processAction({
+  questionResponse,
+  interactionStep,
+  campaignContactId,
+  contact,
+  campaign,
+  organization,
+  previousValue
+}) {
   try {
-    throw new Error("zapier-action.processAction is not implemented");
+
+    const baseUrl = getConfig("BASE_URL", organization);
+    const conversationLink = `${baseUrl}/app/${organization.id}/todos/review/${contact.id}`;
+    const payload = {
+      questionResponse: questionResponse,
+      interactionStep: interactionStep,
+      campaignContactId: campaignContactId,
+      contact: contact,
+      campaign: campaign,
+      organization: organization,
+      previousValue: previousValue,
+      conversationLink: conversationLink
+    };
+
+    const stringifiedPayload = JSON.stringify(payload);
+
+    const url = getConfig("ZAPIER_WEBHOOK_URL", organization);
+    const zap_timeout = getConfig("ZAPIER_TIMEOUT_MS", organization) || 5000;
+    log.info(`Zapier timeout: ${zap_timeout}`);
+
+    var final_url = "";
+    const answer_option = interactionStep.answer_option;
+
+    const config = JSON.parse(getConfig("ZAPIER_CONFIG_OBJECT", organization)) || {};
```

Comment:
this will fail when ZAPIER_CONFIG_OBJECT doesn't exist so let's do:
`const config = JSON.parse(getConfig("ZAPIER_CONFIG_OBJECT", organization) || "{}");`